cfgov/unprocessed/css/blog-docs.less

```diff
@@ -420,13 +420,19 @@
     }
 }
 
-.filtered-by {
+.filtered-by, .error-container {
     .cf-notification  {
         display: block;
         margin: @gutter__em 0 @gutter__em * 2;
     }
 }
 
+.error-container {
+    .cf-notification  {
```

Comment:
I can see this was existing on L:424, but is there a way `cf-notification` can not be included here or can be added as a notification modifier? Overriding notification styles outside of the main notification style declaration area won't work well in atomic land. Also, notification shouldn't know it has a margin in atomic land so this may be moot after https://github.com/cfpb/cfgov-refresh/pull/1202